PAY-6182: Adapt docs to Payment Services drop-in V2 (#433)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/payment-services/installation.mdx

description: Learn how to install the Payment Services drop-in component on your site.
---

import Aside from '@components/Aside.astro';
import Badge from '@components/overrides/Badge.astro';
import Link from '@components/Link.astro';
import Task from '@components/Task.astro';
import Tasks from '@components/Tasks.astro';

> This guide will help you install and configure the Payment Services drop-in component in your storefront.

Reviewer: ```suggestion
This guide explains how to install and configure the Payment Services drop-in component in your storefront.
```

## Payment Services extension <Badge tooltip="Applies to Adobe Commerce on Cloud projects (Adobe-managed PaaS infrastructure) and on-premises projects only." text="PaaS only" variant="note" href="https://experienceleague.adobe.com/en/docs/commerce/user-guides/product-solutions" />

> The Payment Services drop-in component requires version <Link href="https://experienceleague.adobe.com/en/docs/commerce/payment-services/release-notes#v2100" text="2.10.0" /> or higher of the Payment Services extension.

Reviewer: ```suggestion
The Payment Services drop-in requires the Payment Services extension version <Link href="https://experienceleague.adobe.com/en/docs/commerce/payment-services/release-notes#v2100" text="2.10.0" /> or higher.
```

## Onboard to Payment Services

> Before you can use the Payment Services component on your storefront, you must <Link href="https://experienceleague.adobe.com/en/docs/commerce/payment-services/get-started/onboard" text="onboard" /> to Payment Services in the Adobe Commerce Admin.

Reviewer: ```suggestion
Before you can use the Payment Services component in your storefront, you must <Link href="https://experienceleague.adobe.com/en/docs/commerce/payment-services/get-started/onboard" text="onboard" /> to Payment Services in the Adobe Commerce Admin.
```

## Apple Pay domain verification file <Badge tooltip="Applies to Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer projects only (Adobe-managed SaaS infrastructure)." text="SaaS only" variant="success" href="https://experienceleague.adobe.com/en/docs/commerce/user-guides/product-solutions" />

> The following steps show how to register your storefront domain with Apple, which is required for using Apple Pay.

Reviewer: ```suggestion
The following steps explain how to register your storefront domain with Apple, which is required to use Apple Pay.
```

<Tasks>
  <Task>
    ### Download the file
    Download the Apple Pay domain verification file.
    <Link href="https://paypalobjects.com/devdoc/apple-pay/well-known/apple-developer-merchantid-domain-association" text="Click to download" />.
  </Task>

  <Task>
    ### Serve as binary content

>     Add the file to the following path in your storefront repository. Note the added `.bin` extension, which (as of today) is the only way we have in EDS to serve binary content.

Reviewer: ```suggestion
Add the file to the following path in your storefront repository. Note the added `.bin` extension, which is currently the only way we have in EDS to serve binary content.
```

    ```txt showLineNumbers=false
    /.well-known/apple-developer-merchantid-domain-association.bin
    ```
  </Task>

  <Task>
    ### Register sandbox domain

>     Add a <Link href="https://www.aem.live/docs/redirects" text="redirect" /> to make the domain verification file accessible on https://your-sandbox-domain.com/.well-known/apple-developer-merchantid-domain-association. Once the redirect is in place, contact your sales representative to register the sandbox domain with Apple.

Reviewer: ```suggestion
Add a <Link href="https://www.aem.live/docs/redirects" text="redirect" /> to make the domain verification file accessible at `https://your-sandbox-domain.com/.well-known/apple-developer-merchantid-domain-association`. Once the redirect is in place, contact your sales representative to register the sandbox domain with Apple.
```

    | Source                                                    | Destination                                                    |
    |-----------------------------------------------------------|----------------------------------------------------------------|
    | .well-known/apple-developer-merchantid-domain-association | /.well-known/apple-developer-merchantid-domain-association.bin |

    <Aside type="caution" title="Sandbox only">

>       Serving the file behind a redirect works for sandbox domain registration but not for production. For production, the file must be served directly, without a redirect.

Reviewer: ```suggestion
Serving the file via a redirect works for sandbox domain registration, but not for production. In production, the file must be served directly, without a redirect.
```

    </Aside>
  </Task>

  <Task>
    ### Register production domain

>     Set up a <Link href="/setup/configuration/content-delivery-network/#url-rewrites" text="URL rewrite" /> at the CDN level to make the domain verification file directly accessible on https://your-production-domain.com/.well-known/apple-developer-merchantid-domain-association, without redirects. Once the URL rewrite is in place, contact your sales representative to register the production domain with Apple.

Reviewer: ```suggestion
Set up a <Link href="/setup/configuration/content-delivery-network/#url-rewrites" text="URL rewrite" /> at the CDN level to make the domain verification file directly accessible at https://your-production-domain.com/.well-known/apple-developer-merchantid-domain-association, without redirects. After the rewrite is in place, contact your sales representative to register the production domain with Apple.
```
